tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/memory/MemorySkewJoinSuite.scala

```diff
@@ -100,8 +101,8 @@ class MemorySkewJoinSuite extends AnyFunSuite
       })
       .toDF("key", "fa", "fb", "fc", "fd")
     df2.createOrReplaceTempView("view2")
-    new File("./df1").delete()
-    new File("./df2").delete()
+    JavaUtils.deleteRecursively(new File("./df1"))
+    JavaUtils.deleteRecursively(new File("./df2"))
```

Comment:
This is unrelated to the PR - but was failing on my local box, and so I addressed it.